src/pages/index.js

```diff
@@ -10,31 +11,70 @@ import {
   canSeeOperativeWorkOrders,
 } from '@/utils/userPermissions'
 import { getQueryProps } from '@/utils/helpers/serverSideProps'
+import Search from '../components/Search/Search'
+import CurrentUserWrapper from '../components/WorkOrders/CurrentUserWrapper'
+import MobileWorkingWorkOrdersView from '../components/WorkOrders/MobileWorkingWorkOrdersView/MobileWorkingWorkOrdersView'
+import WorkOrdersView from '@/components/WorkOrders/WorkOrdersView'
+import NewTabs from '../components/NewTabs/Index'
 
 const Home = ({ query }) => {
   const { user } = useContext(UserContext)
+  const router = useRouter()
 
   const [loading, setLoading] = useState(true)
 
+  const titles = ['Current Work Orders', 'Past Work Orders']
+
+  const handleTabClick = (index) => {
+    index === 1 && router.push('/oldjobs')
+  }
+
+  const ariaSelected = useMemo(() => {
+    return router.pathname === '/oldjobs' ? 1 : 0
+  }, [router.pathname])
+
   const HomeView = () => {
     if (user && canSeeWorkOrders(user)) {
       // Use saved filter preset in local storage as the default applied filters (if present)
       const defaultFilters = JSON.parse(
         localStorage.getItem('RH - default work order filters')
       )
-
       if (Object.entries(query).length === 0) {
         return (
-          <WorkOrdersView
-            pageNumber={1}
-            {...(defaultFilters && { query: defaultFilters })}
-          />
+          <>
+            <NewTabs
+              titles={titles}
+              onTabChange={handleTabClick}
+              ariaSelected={ariaSelected}
+            />
+            <WorkOrdersView
+              pageNumber={1}
+              {...(defaultFilters && { query: defaultFilters })}
+            />
+          </>
         )
       } else {
-        return <WorkOrdersView query={query} />
+        return (
+          <>
+            <WorkOrdersView query={query} />
+          </>
+        )
       }
     } else if (user && canSeeOperativeWorkOrders(user)) {
-      return <MobileWorkingWorkOrdersView />
+      return (
+        <>
+          <NewTabs
+            titles={titles}
+            onTabChange={handleTabClick}
+            ariaSelected={ariaSelected}
+          />
+          <CurrentUserWrapper>
+            {({ currentUser }) => (
+              <MobileWorkingWorkOrdersView currentUser={currentUser} />
+            )}
+          </CurrentUserWrapper>
+        </>
```

Comment:
To keep this page consistant, would it be be better to keep this logic within the MobileWorkingWorkOrdersView component? There is a lot going on here, that could be abstracted away.

Reply:
Try to make this simplified. Move tabs into MobileWorkingWorkOrdersView